Automattic/wp-calypso
# Update/checkout plan messaging

#### Changes proposed in this Pull Request

This PR cleans up some messaging in checkout so that it doesn't clutter the interface with too much copy.

**Before**
![image](https://user-images.githubusercontent.com/6981253/98580525-181a2880-228e-11eb-88ca-bef145980d9f.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/98580424-ee610180-228d-11eb-86dd-510a93c1fa37.png)


#### Testing instructions



* Upgrade a paid plan and confirm the "Discount for first year" messaging shows.
* Upgrade a paid plan purchased with a two year term and confirm "Discount for first term" appears. 
* Add a plan to a free site and confirm this messaging doesn't appear.
